client/client_state.h

```diff
@@ -521,6 +519,24 @@ struct CLIENT_STATE {
 #endif
 
     KEYWORDS keywords;
+
+    double current_cpu_usage_limit() {
```

Comment:
Should this function be moved to the .cpp file?

Reply:
I like to put short functions in the .h file.